cohesity_management_sdk/models/azure_protection_source.py

```diff
@@ -197,17 +216,24 @@ def from_dictionary(cls,
         name = dictionary.get('name')
         num_cores = dictionary.get('numCores')
         physical_source_id = dictionary.get('physicalSourceId')
+        region = dictionary.get('region')
         resource_id = dictionary.get('resourceId')
         restore_task_id = dictionary.get('restoreTaskId')
         subscription_id = dictionary.get('subscriptionId')
         subscription_type = dictionary.get('subscriptionType')
+        tag_attributes = None
```

Comment:
why is this handled differently than other variables ?

Reply:
value for **tag_attributes** field is list of objects, so here we are iterating through the list of objects.
```
tag_attributes = None
if dictionary.get('tag_attributes')!= None:
            tag_attributes = list()
            for structure in dictionary.get('tag_attributes'):
                tag_attributes.append(cohesity_management_sdk.models.tag_attribute.TagAttribute.from_dictionary(structure))
```